src/Mod/Sketcher/Gui/ViewProviderSketch.cpp

```diff
@@ -970,8 +970,10 @@ bool ViewProviderSketch::mouseButtonPressed(int Button, bool pressed, const SbVe
     // Radius maximum to allow double click event
     const int dblClickRadius = 5;
 
-    double x, y;
+    double x = NAN;
+    double y = NAN;
```

Comment:
These should probably be:
```suggestion
    double x = std::numeric_limits<double>::quiet_NaN();
    double y = std::numeric_limits<double>::quiet_NaN();
```
if NaN is really the best initialization value.